var/spack/repos/builtin/packages/lzo/package.py

```diff
@@ -37,13 +37,14 @@ class Lzo(Package):
     version('2.06', '95380bd4081f85ef08c5209f4107e9f8')
     version('2.05', 'c67cda5fa191bab761c7cb06fe091e36')
 
-    def install(self, spec, prefix):
-        configure_args = [
-            '--prefix={0}'.format(prefix),
+    def configure_args(self):
+        return [
             '--disable-dependency-tracking',
             '--enable-shared'
         ]
-        configure(*configure_args)
-        make()
+
+    @AutotoolsPackage.sanity_check('build')
+    @AutotoolsPackage.on_package_attributes(run_tests=True)
+    def check(self):
```

Comment:
neat! 👍 

Whether or not `make check` and `make test` go to the base class, supposedly, depends on how many packages currently do this or support this in principle. If not too many, then it would be easier to define `check()` there with `make(check)`.

If we have default implementation of `check()` with `make('check')`, then supposedly one would have to override it in each package which does not have these targets.